cmd/ttn-lw-cli/internal/util/flags.go

```diff
@@ -86,6 +101,22 @@ func UpdateFieldMask(cmdFlags *pflag.FlagSet, fieldMaskFlags ...*pflag.FlagSet)
 	return
 }
 
+func DropForbiddenFieldMaskPaths(paths []string, forbiddenPaths []string) []string {
```

Comment:
Use `ttnpb.ExcludeFields()`

Reply:
Also:

- Mutating `paths` isn't what you'd expect from the function signature
- This func misses documentation

Reply:
Switched to the existing `ttnpb.ExcludeFields()` function